modules/zstr/1.0.7/presubmit.yml

```
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 8.x
+  - 7.x
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+    - '@zstr//...'
```

```suggestion
    - '@zstr//...'
  build_flags:
    - --features=parse_headers
    - --process_headers_in_dependencies
```
Provides some CI coverage for this header-only library.

---

BcrValidationResult.FAILED: Missing bazel version for task 'build_flags' in the presubmit.yml file.

please advise

---

Done